t/t2111-job-ingest-config.t

```diff
@@ -45,6 +45,30 @@ test_expect_success 'job-ingest: configuration can be reloaded' '
 test_expect_success 'job-ingest: verify that feasibility plugin is in effect' '
 	test_must_fail flux submit -n 1024 hostname
 '
```

Comment:
commit title: "testsuite: test configuraiton of ingest.buffer-size": s/configuraiton/configuration/

Reply:
Thanks! Fixed that commit message and will set MWP.